pkg/mimirpb/compat.go

```diff
@@ -186,6 +221,14 @@ func FromHistogramProtoToFloatHistogram(hp Histogram) *histogram.FloatHistogram
 		NegativeBuckets:  hp.GetNegativeCounts(),
 	}
 }
+
+func FromHistogramProtoToPromCommonHistogram(h *Histogram) *model.SampleHistogram {
```

Comment:
[nit] As I suggested in another PR I think we should remove the "common" part from these function names. "prom" is enough to identify the model we're converting to/from.

Reply:
fixed